lib/include/hse_ikvdb/kvdb_cparams.h

```diff
@@ -35,6 +35,12 @@ kvdb_cparams_get(
     size_t                     buf_sz,
     size_t *                   needed_sz);
 
+merr_t
+kvdb_cparams_set(
```

Comment:
This function isn't really necessary since you cannot set a cparam. Can remove if you all think it is worthless. It was essentially free to implement.